connectors/sources/postgresql.py

```diff
@@ -300,6 +311,7 @@ async def data_streamer(
             list: It will first yield the column names, then data in each row
         """
         if query is None and row_count is not None and order_by_columns is not None:
+            self._logger.debug(f"Streaming records from database using query: {query}")
```

Comment:
This seems wrong. `query is None` was just asserted above this

Reply:
I think the logs are swapped for if-else conditions.